data.json

```diff
@@ -863,4 +863,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/71710239/117578918-55148a00-b10e-11eb-93f4-8d2bd01dcd8c.png",
     "issueNumber": 487
   }
```

Comment:
Add `,` to the above bracket and please remove the screenshot in PR. GitHub actions will do that from your issue itself